internal/rest/avi_obj_vsvip.go

```diff
@@ -646,6 +665,9 @@ func networkNamesToVips(vipNetworks []akov1alpha1.AviInfraSettingVipNetwork, ena
 		}
 		newVip.SubnetUUID = proto.String(vipNetwork.NetworkName)
 		vipList = append(vipList, newVip)
+		if vipNetwork.V6Cidr != "" {
```

Comment:
Did you test in cloud env? Can you please update your testing?

Reply:
only vCenter cloud is supported